### PR TITLE
Recommend harmonic for gazebo version

### PR DESCRIPTION
**Pull request by Ryanf55**
Recommend harmonic for the gazebo version.

Depends on:
* https://github.com/ArduPilot/ardupilot_gz/pull/54
* https://github.com/ArduPilot/ardupilot_gazebo/pull/93

### CODE REVIEW

File: dev/source/docs/ros2-gazebo.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ ROS 2 with Gazebo
 
 Once ROS2 is correctly :ref:`installed <ros2>` and running :ref:`SITL <ros2-sitl>`, we can integrate ArduPilot with Gazebo. 
 
-First, install `Gazebo Garden <https://gazebosim.org/docs/garden/install>`__ or `Gazebo Harmonic <https://gazebosim.org/docs/harmonic/install>`__
+First, install `Gazebo Harmonic (recommneded) <https://gazebosim.org/docs/harmonic/install>`__ or `Gazebo Garden <https://gazebosim.org/docs/garden/install>`__.
```
Comment (Ryanf55) [inline dev/source/docs/ros2-gazebo.rst:9]:
```suggestion
First, install `Gazebo Harmonic (recommended) <https://gazebosim.org/docs/harmonic/install>`__ or `Gazebo Garden <https://gazebosim.org/docs/garden/install>`__.
```

**Review by srmainwaring — APPROVED** [review]

**Comment by Hwurzburg** [comment]
thnanks

**Comment by Ryanf55** [comment]
> thnanks..still marked as draft

Yes, it will be a draft till the upstream PR's merge. When it is out of draft, it will be ready for merge.

**Comment by Hwurzburg** [comment]
tnx

**Pull request merged**